packages/merge-styles/etc/merge-styles.api.md

```diff
@@ -4,29 +4,28 @@
 
 ```ts
 
+// Warning: (ae-forgotten-export) The symbol "ObjectOnly" needs to be exported by the entry point index.d.ts

```

Comment:
1. can you add in PR description why this change is needed? what scenario was failing with which change made?

Reply:
Think about this case:

```tsx
const result = concatStyleSets({ a: { ... } } , undefined)
```

What is the type of result? It would be:

```tsx
{ a: { ... } } & undefined
```

...Which results in `never`. Which is not quite what we're looking for. The `ObjectOnly` type ensures that the undefined does not turn the intersection into a `never` and returns at least an object definition in response.